javascript/javascript1/week3/NOnoN0nOYes (Note taking app).js

```diff
@@ -0,0 +1,56 @@
+const notes = [];
+
+function saveNote(content, id) {
```

Comment:
what output will return?  saveNote(10, "Car wash");  saveNote(11, 11);  saveNote("Play with kids", "Car wash"); I suggest before push value in object, try to validate value by type of.